components/coachmark/index.css

```diff
@@ -18,8 +18,8 @@
 	/* @passthrough start */
 	--mod-buttongroup-justify-content: flex-end;
 	--mod-popover-border-width: var(--spectrum-border-width-100);
-	--mod-popover-corner-radius: var(--spectrum-corner-radius-100);
-	--mod-popover-content-area-spacing-vertical: 0;
+	--mod-popover-corner-radius: var(--spectrum-corner-radius-large-default)
+	--mod-popover-content-area-spacing: 0;
```

Comment:
I updated this mod name because I changed popover-content-area-spacing in the popover CSS- there's now padding on all sides of popover instead of just on the top & bottom. However, I am unsure if this is actually the correct styling for the popovers in coachmark. Is this something to address in this PR, or in the coachmark PR #3412?